content/200-concepts/100-components/02-prisma-client/064-custom-models.mdx

```diff
@@ -6,11 +6,95 @@ metaDescription: 'This page explains how to wrap Prisma Client in custom models'
 
 <TopBlock>
 
-As your application grows, you may find the need to group related logic together. We suggest either wrapping a model in a class or extending Prisma Client model object.
+As your application grows, you may find the need to group related logic together. We suggest either:
+
+- Creating static methods using a [Prisma Client extension](/concepts/components/prisma-client/client-extensions)
+- Wrapping a model in a class
+- Extending Prisma Client model object
 
 </TopBlock>
 
-## Wrap a Prisma Model in a Class
+## Static methods with Prisma Client extensions
```

Comment:
Would it be worth adding  [instance methods](https://www.prisma.io/blog/client-extensions-preview-8t3w27xkrxxn#example-instance-methods) to this as well? Would you consider this part of "customizing models"?

Reply:
The instance method Prisma Client extensions example documents a hybrid of the repository and unit of work patterns used by tools such as Ruby on Rails and Entity Framework.

We could look at this more holistically, focus the information on querying patterns, and create a page that documents that instead.